root/static/scripts/common/utility/reactTextContent.js

```diff
@@ -0,0 +1,25 @@
+// This file is part of MusicBrainz, the open internet music database.
+// Copyright (C) 2015 MetaBrainz Foundation
+// Licensed under the GPL version 2, or (at your option) any later version:
+// http://www.gnu.org/licenses/gpl-2.0.txt
+
+const nonEmpty = require('./nonEmpty');
+
+function reactTextContent(reactElement, previous = '') {
```

Comment:
Yeah, as the link shows it's only really used for artists (artist credits) and recordings (which are linked using track names on release pages) right now.

Usually the name variation is just gonna be a string in those cases. But I think we pass in HTML in the edit diff display stuff. Which hasn't been converted to React yet.